src/v4/htype.rs

```diff
@@ -51,20 +53,36 @@ pub enum HType {
     MilStd188220,
     /// 23 Metricom
     Metricom,
+    /// 24 IEEE1394.1995
+    IEEE13941995,
```

Comment:
Let's just add these new variants to the v6 type if you don't mind. It's a breaking change to add them here.